### PR TITLE
Fixes #33116 - Add/Remove package Group Filters

### PR DESCRIPTION
**Pull request by Andrewgdewar**
- [x] Add/Remove actions working
- [x] Need to update repositories repo with same checked/unchecked logic.
- [x] Needs Tests

### CODE REVIEW

File: webpack/scenes/ContentViews/ContentViewsConstants.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ export const PUBLISH_CONTENT_VIEW_KEY = 'CONTENT_VIEW_PUBLISH';
 export const UPDATE_CONTENT_VIEW = 'UPDATE_CONTENT_VIEW';
 export const UPDATE_CONTENT_VIEW_SUCCESS = 'UPDATE_CONTENT_VIEW_SUCCESS';
 export const UPDATE_CONTENT_VIEW_FAILURE = 'UPDATE_CONTENT_VIEW_FAILURE';
+export const ADD_CONTENT_VIEW_PACKAGE_FILTER_KEY = 'ADD_CONTENT_VIEW_PACKAGE_FILTER_KEY';
+export const DELETE_CONTENT_VIEW_PACKAGE_FILTER_KEY = 'DELETE_CONTENT_VIEW_PACKAGE_FILTER_KEY';
 export const cvDetailsKey = cvId => `${CONTENT_VIEWS_KEY}_${cvId}`;
```
Comment (sjha4) [inline webpack/scenes/ContentViews/ContentViewsConstants.js:15]: ADD_CONTENT_VIEW_PACKAGE_GROUP_FILTER_RULE or if that seems too long ADD_CV_PACKAGE_GROUP_FILTER_RULE.. I like the 2nd one better..

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/ContentViewsConstants.js:15]: :+1:

File: webpack/scenes/ContentViews/Details/ContentViewDetailActions.js [hunk 2]
```diff
@@ -152,6 +154,26 @@ export const getCVFilterPackageGroups = (cvId, filterId, params) => get({
   url: api.getApiUrl('/package_groups'),
 });
 
+export const addCVFilterPackageRule = (filterId, params, handleSuccess) => post({
+  type: API_OPERATIONS.POST,
+  key: ADD_CONTENT_VIEW_PACKAGE_GROUP_FILTER_RULE,
+  url: api.getApiUrl(`/content_view_filters/${filterId}/rules`),
+  params,
+  handleSuccess,
+  successToast: () => __('Filter added'),
```
Comment (sjha4) [inline webpack/scenes/ContentViews/Details/ContentViewDetailActions.js:163]: Thoughts on having something like: "Package group added to filter" and similar for error toast.

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/ContentViewDetailActions.js:163]: Yeah, that was an oversight, good catch.
Would Package group rule added to filter make sense?

**Review by sjha4 — APPROVED** [review]
Code LGTM! Love that bulk buttons are there just waiting for the API to be built.. 😄 
Works great.. APJ!! 🚀

**Pull request merged**